examples/generative/corrdiff/generate.py

```diff
@@ -86,6 +85,10 @@ def main(cfg: DictConfig) -> None:
     out_channels = getattr(cfg, "out_channels", [0, 17, 18, 19])
     img_shape_x = getattr(cfg, "img_shape_x", 448)
     img_shape_y = getattr(cfg, "img_shape_y", 448)
+    patch_shape_x = getattr(cfg, "patch_shape_x", 448)
```

Comment:
I wonder if we need to add a shape check for patches. Patch shapes cannot be any arbitrary number, right?

Reply:
A shape check for patches has been implemented, but the current check may not cover all possible scenarios comprehensively.